.changeset/sour-fans-type.md

```diff
@@ -0,0 +1,24 @@
+---
+'braid-design-system': minor
+---
+
+---
+updated:
+  - Disclosure
+---
+
+**Disclosure:** Add inline content support
+
+Provides support for using a Disclosure in the context of sentence, by allowing it to be nested within a typographic component, i.e. `Text`. This enables the call to action to sit inline with other copy.
```

Comment:
I don't think the comma is needed.

```suggestion
Provides support for using a Disclosure in the context of a sentence by allowing it to be nested within a typographic component, i.e. `Text`. This enables the call to action to sit inline with other copy.
```

Also more broadly, what does `in the context of a sentence` even mean? It honestly sounds more complicated than it needs to be. Feel like it could be simplified to `inline with other content` or something.

Reply:
I agree actually, although to avoid confusion with the behaviour of our `Inline` layout component I'm thinking something more like "within a sentence" or similar.
```suggestion
Provides support for using a Disclosure within a sentence by allowing it to be nested within a typographic component, i.e. `Text`.
```